at-dungnguyen/Week3/2018_07_05_at_dungnguyen3_excercise_day4_3.rb

```diff
@@ -0,0 +1,32 @@
+class HtmlToText
+  def initialize(html)
+    @html = html
+  end
+
+  def get_html(html)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/f0d0660bf5f99a9a983d87c7561868b11394b606/at-dungnguyen/Week3/2018_07_05_at_dungnguyen3_excercise_day4_3.rb#L6"></span>Metrics/PerceivedComplexity: Perceived complexity for get_html is too high. [10/8]</td>
    </tr>
  </tbody>
</table>